### ISSUE TITLE
Error building docker image - install-mamba.sh

### ISSUE THREAD

**Issue by palmeida** [openwpm/OpenWPM#1108]
I'm getting an error building the docker image after cloning the repository:

`$ docker build -f Dockerfile -t openwpm .`

> Extracting mamba-1.5.9-py312h9460a1c_0.conda
!!!!!! Mambaforge is now deprecated !!!!!
Future Miniforge releases will NOT build Mambaforge installers.
We advise you switch to Miniforge at your earliest convenience.
More details at https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/.
If you are unable to switch to Miniforge, you may pin your installer version to one found in 
https://github.com/conda-forge/miniforge/releases/tag/24.3.0-1
or if you lack the system requirements (Linux glibc >= 2.17, or macOS + x86-64bit >= 10.13)
you may pin your installer to one older version found in 
https://github.com/conda-forge/miniforge/releases/tag/24.3.0-0
This Miniforge installer will ceese to work in 2025.
If you require this Mambaforge installer you may pin to 
https://github.com/conda-forge/miniforge/releases/tag/24.7.1-0
ERROR: executing pre_install.sh failed
./install-mamba.sh: line 14: /opt/mamba/etc/profile.d/conda.sh: No such file or directory
./install-mamba.sh: line 16: conda: command not found
./install-mamba.sh: line 17: conda: command not found
./install-mamba.sh: line 18: conda: command not found
The command '/bin/bash -c ./install-mamba.sh' returned a non-zero code: 127

I tried just [switching to Miniforge](https://github.com/conda-forge/miniforge?tab=readme-ov-file#install) and adjusting the paths in the Dockerfile and it worked.

**Comment by vringar** [comment]
Hey,
thanks for discovering the issue and fixing it!
Could you open a PR with the changes you made?

**Comment by englehardt** [comment]
Should be fixed by https://github.com/openwpm/OpenWPM/pull/1109

**Comment by vringar** [comment]
Can confirm that it is fixed on main :)

**Comment by palmeida** [comment]
Yes, thank you! I completely forgot about this, but englehardt was much more thorough in replacing all mamba references than I had been when I solved my problem locally, which was why I didn't immediately do a pull request back then.
